DFC.App.RelatedCareers/Controllers/SegmentController.cs

```diff
@@ -19,8 +20,10 @@ public class SegmentController : Controller
         private const string IndexActionName = nameof(Index);
         private const string DocumentActionName = nameof(Document);
         private const string BodyActionName = nameof(Body);
-        private const string SaveActionName = nameof(Save);
+        private const string PostActionName = nameof(Post);
+        private const string PutActionName = nameof(Put);
         private const string DeleteActionName = nameof(Delete);
+        private const string PatchRelatedCareersActionName = nameof(PatchRelatedCareersData);
```

Comment:
Doesn't make sense to have the nameof stored as const, why can't we use `nameof(!)` as is?

Reply:
Its done multiple times so makes sense to have as a variable